CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# POA Network
+
+## To Contribute
+1. Fork the repository
+https://github.com/poanetwork/hbbft
+2. Create a feature branch
+3. Write tests to cover the work 
+4. Commit changes
+5. Push to the branch
+6. Create a new pull request following the PR protocol below
+
+## Pull Request (PR) Protocol
+
+All pull requests must include: 
+* A clear, readable description of the purpose of the PR
+* A clear, readable description of changes
+* A title that includes (Fix), (Feature), or (Refactor)
+**example:** (Fix) price of 1 token in Wei > 18 decimals
+* A single commit message for one specific fix or feature. A separate PR should be made for each specific change.
+* Any additional concerns or comments (optional)
+
+All accepted and completed PRs are updated in the Wiki documentation.
```

Comment:
Do we already have a Wiki entry for hbbft? I'd only add this sentence once we do.

(Also: _Should_ we have one? If we start duplicating the documentation, it will start to go out of sync.)